nsxt/resource_nsxt_policy_ipsec_vpn_local_endpoint.go

```diff
@@ -354,3 +358,20 @@ func nsxtVPNServiceResourceImporter(d *schema.ResourceData, m interface{}) ([]*s
 
 	return []*schema.ResourceData{d}, nil
 }
+
+func validateIPv6LocaleServiceConflict(ctx context.Context, diff *schema.ResourceDiff, meta interface{}) error {
+	return validateIPv6WithLocaleService(diff, "local_address", "validateIPv6LocaleServiceConflict")
+}
+
+func validateIPv6WithLocaleService(diff *schema.ResourceDiff, fieldName, logPrefix string) error {
+	address := diff.Get(fieldName).(string)
+	servicePath := diff.Get("service_path").(string)
+	ip := net.ParseIP(address)
+	isIPv6 := ip != nil && strings.Contains(address, ":") && ip.To4() == nil
```

Comment:
This is good, but maybe just one of `strings.Contains(address, ":")` and `ip.To4() == nil` would have been enough?